tests/test_courses.py

```diff
@@ -2,7 +2,9 @@
 from dotenv import load_dotenv
 import pytest
 
+from requests.exceptions import RequestException
 from sylveon._classes._connection import GSConnection
+from sylveon._classes._courses import Course
```

Comment:
```suggestion
from sylveon._classes._connection import GSConnection
```
I think these imports can be removed since they aren't used